### DISCUSSION TITLE
Trying to download pdf using DOMPDF but getting Service Unavailable error

### DISCUSSION THREAD

**Discussion by SoftwareSamadhan** [dompdf/dompdf#3368] [Q&A]
Hii,
I have multiple projects running in Godaddy cpanel shared hosting. Everything was okay yesterday. When trying to download a generated pdf (DOMPDF), every code is running well but the download code i.e.
 `$pdf->download('filename.pdf');` is giving an error-

> Service Unavailable
The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.
Additionally, a 503 Service Unavailable error was encountered while trying to use an ErrorDocument to handle the request.

The same project is running well on my local machine and in a different Godaddy account account. Can anyone please help? It is so urgent for me.

Is there any cache I need to delete? for which the generated PDF is unable to be downloaded.

**Comment by bsweeney** [comment]
Service unavailable means the web server did not respond to the request in a timely manner. My guess is that Dompdf is taking too long to render the PDF and the timeout for response is reached. If you can list any PHP errors in your logs that could help determine the issue. If not, it might help to see a sample of the HTML to see if there are any obvious issues that might cause an extraordinary amount of time rendering the document.
